ios/ElectrodeApiImpl/APIImpls/Core/ENCoreDelegate.swift

```diff
@@ -53,4 +57,8 @@ import UIKit
         }
         return nil
     }
-}
+
+    func reloadeView(viewController: UIViewController) {
+        self.loadView(viewController: viewController)
+    }
+}
```

Comment:
Don't remove the newline character on the last line please ;)

Reply:
you have an extra 'e' in method name (reloadeView)